### ISSUE TITLE
Move everything in Docs out of Getting Started

### ISSUE THREAD

**Issue by oxinabox** [JuliaPlots/PlotDocs.jl#71]
from https://github.com/JuliaPlots/Plots.jl/issues/1369

Getting Started contains way too much.
Most projects have just Install instructions and some basic example/tutorial.

It actually took me over a year to realize that there was an attributes page, because I never read the getting started, since i got started I think before their were docs.

I suggest the following structure for the side bar:

 - Home
 - Learning
 - Contributing 
 - Ecosystem 
 - Getting Started
	 - Installation
	 - Basics
	 - Tutorial
 - Manual
	 - Input Data
	 - Output
	 - Layouts
	 - Attributes
	 - Colors
	 - Animations
	 - Backends
	 - Supported Attributes
 - Advanced Topics
	 - Internals
	 - Recipes
 - Examples
	 - PyPlot
	 - GR
	 - PlotlyJS
	 - PGFPlots
	 - UnicodePlots
	 - InspectDR

In short it is basically move Tutorial into "Getting Started",
and move everything else into "Manual",
and reorder things.
Delete 0.7.

**Comment by oxinabox** [comment]
@mkborregaard : 
> I agree, except that I think we should delete Plots v0.7, move Layouts and Recipes to Manual, and just keep Internals and pipeline under "Advanced Topics". (I also want the Ecosystem page to change, and StatPlots and PlotRecipes readmes to be moved to separate tabs).
This should really be an issue on PlotDocs, though.

Layout definitely should be under Manual.
idk about Recipes, I'm still scared of them.
Maybe for that reason alone that need to move.

There is no pipeline page on the website right now.

**Comment by mkborregaard** [comment]
Hear, hear. 👍 from me

**Comment by deszoeke** [comment]
I understand [Attributes](http://docs.juliaplots.org/latest/attributes/) is the authoritative page to look up all the attributes of different plot/series types in Plots. It starts off intimidatingly with occult features and inner workings like Aliases and Magic Arguments. These are enablers for how Plots works, but they're not concepts that a new user needs yet. First time users need to learn how to do something one robust way, then only later learn how efficiently they can use aliases and magic.

I think the diction could be more precise. e.g. What's an Argument, how does it compare to an Attribute?

**Comment by mkborregaard** [comment]
Good point.

**Comment by daschw** [comment]
This was fixed in #104
